extensions/Worker.Extensions.SignalRService/src/SignalRConnectionInfoInputAttribute.cs

```diff
@@ -1,16 +1,22 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-﻿using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
-
 namespace Microsoft.Azure.Functions.Worker
 {
-    public sealed class SignalRConnectionInfoInputAttribute : InputBindingAttribute
+    /// <summary>
+    /// Attribute used to bind necessary information for a SignalR client to connect to SignalR Service.
+    /// <para> 
+    /// The connection info object will have the following properties:
+    /// <code>
+    /// public class MyConnectionInfo 
+    /// {
+    /// public string Url { get; set; }
+    /// public string AccessToken { get; set; }
+    /// } 
+    /// </code>
+    /// </para>
+    /// </summary>
+    public sealed class SignalRConnectionInfoInputAttribute : SignalRNegotiationBaseAttribute
     {
```

Comment:
This is introducing a breaking change in a public type in a minor version. Intentional? Seems like a local refactor based on the other attribute that was introduced.

Reply:
According to doc [Types of breaking changes - .NET](https://docs.microsoft.com/en-us/dotnet/core/compatibility/), this seems not to be a breaking change. But anyway, I recovered it. 
> ALLOWED: Moving a member into a class higher in the hierarchy than the type from which it was removed